src/features/device-state/state-patch-utils.ts

```diff
@@ -39,6 +39,8 @@ export const v2ValidPatchFields: Array<
 	'download_progress',
 ];
 
+export const macIpAddressesTruncationLength = 255;
```

Comment:
How about exporting a function rather than the length?
It could also be doing a `['ip_address', 'mac_address'].forEach(...)`.
Or it could be made even more generic like `truncateShortTextFields<T extends object>(object: T, keys: Array<keyof T>)`, which could allow us to use it in more place (eg maybe for metrics).

Reply:
Made slightly more generic with https://github.com/balena-io/open-balena-api/commit/dd736ce43d85bc640f9b08a479ed783f0f56ed9d